docker-compose.yml

```diff
@@ -161,38 +174,14 @@ services:
             - "./docs:/srv/app"
         ports:
             - "${EXPOSED_DOCS_PORT:-8002}:3000"
+    mailhog:
+        image: mailhog/mailhog
+        ports:
+            - "${EXPOSED_MAILHOG_SMTP_PORT:-1025}:1025"
+            - "${EXPOSED_MAILHOG_WEB_PORT:-8025}:8025"
     consul:
         image: consul:1.7
         healthcheck:
             test: ["CMD", "sh", "-c", "curl --fail http://localhost:8500 || exit 1"]
```

Comment:
In this time we use `consul` only to handle `rabbitmq` in 3 nodes. So if we start use only one `rabbitmq`, we can erase `consul` from this stack

Reply:
We can't remove consul, because whole configuration is based on Consul with RabbitMQ. When i remove consul i must change RabbitMQ configuration file in `config`. This change will destroy other environments like production.